app/components/have-a-question.js

```diff
@@ -0,0 +1,131 @@
+import React, { useState, useEffect, useRef } from 'react'
+import { createUseStyles } from 'react-jss'
+import TextareaAutosize from 'react-textarea-autosize'
+import cx from 'classnames'
+
+const defaultValue = 'Have a question? Ask away....'
+const buttonMessage = 'Submit'
+const emailMessage = 'Want a reply? Leave your email.'
+export default function HaveAQuestion(props) {
+    const [message, setMessage] = useState(emailMessage)
+
+    const classes = useStyles()
+    const [askEmail, setAskEmail] = useState(false)
+    const [response, setResponse] = useState(null)
+    const [subject, setSubject] = useState(defaultValue)
+    // const [submitButton, setSubmitButton] = useState(false)
+    const { className, style } = props
+
+    useEffect(() => {}, [subject], [message])
+
+    const handleKeyPress = e => {
+        if (e.key === 'Enter') {
+            setAskEmail(true)
+            e.preventDefault()
+        }
+    }
+    const handelInput = e => {
+        setAskEmail(true)
+    }
+
+    const handleKeyPress2 = e => {
+        if (e.key === 'Enter') {
+            e.preventDefault()
+        }
+    }
+    const handelInput2 = e => {
+        setAskEmail(true)
+    }
+
+    const contactUs = e => {
+        let fname = ''
+        let lname = ''
+        let email = ''
+
+        window.socket.emit('send-contact-us', email, fname, lname, subject, message, response => {
+            if (response && response.error) {
+                let { error } = response
+                setResponse(error)
+            } else {
+                setResponse('Your question was sucessfully submitted!')
+                setTimeout(() => 1000)
+            }
+
+            console.log('subject is,', subject)
```

Comment:
Before final check in, the console.log's have to be removed.

If you wanted to be extra fancy, you could put something in your story template that renders the parameters of the emit message in a div after the <HaveAQuestion  component.  You could look at datetime-input.stories.js for a partial example.